README.md

```diff
@@ -68,7 +84,15 @@ metrics.register_callback(function()
 end)
 ```
 
-Using histograms:
+### Histogram
+
+Histogram samples observations and counts them into configurable buckets.
```

Comment:
`Histogram arranges observed data by an array of configurable buckets.` -- something along the lines maybe?

Reply:
Nope, it only stores counts, not actual data

Reply:
done